core/src/test/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunctionTest.java

```diff
@@ -565,6 +565,77 @@ public void exp_missing_value() {
     assertTrue(exp.valueOf(valueEnv()).isMissing());
   }
 
+  /**
+   * Test expm1 with integer value.
+   */
+  @ParameterizedTest(name = "expm1({0})")
+  @ValueSource(ints = {-2, 2})
+  public void expm1_int_value(Integer value) {
+    FunctionExpression expm1 = DSL.expm1(DSL.literal(value));
+    assertThat(
+            expm1.valueOf(valueEnv()),
+            allOf(hasType(DOUBLE), hasValue(1 - Math.expm1(value))));
+    assertEquals(String.format("expm1(%s)", value.toString()), expm1.toString());
+  }
+
+  /**
+   * Test expm1 with long value.
+   */
+  @ParameterizedTest(name = "expm1({0})")
+  @ValueSource(longs = {-2L, 2L})
+  public void expm1_long_value(Long value) {
+    FunctionExpression expm1 = DSL.expm1(DSL.literal(value));
+    assertThat(
+            expm1.valueOf(valueEnv()),
+            allOf(hasType(DOUBLE), hasValue(Math.expm1(value))));
+    assertEquals(String.format("expm1(%s)", value.toString()), expm1.toString());
+  }
+
+  /**
+   * Test expm1 with float value.
+   */
+  @ParameterizedTest(name = "expm1({0})")
+  @ValueSource(floats = {-2F, 2F})
+  public void expm1_float_value(Float value) {
+    FunctionExpression expm1 = DSL.expm1(DSL.literal(value));
+    assertThat(
+            expm1.valueOf(valueEnv()),
+            allOf(hasType(DOUBLE), hasValue(Math.expm1(value))));
+    assertEquals(String.format("expm1(%s)", value.toString()), expm1.toString());
+  }
+
+  /**
+   * Test expm1 with double value.
+   */
+  @ParameterizedTest(name = "expm1({0})")
+  @ValueSource(doubles = {-2D, 2D})
+  public void expm1_double_value(Double value) {
+    FunctionExpression expm1 = DSL.expm1(DSL.literal(value));
+    assertThat(
+            expm1.valueOf(valueEnv()),
+            allOf(hasType(DOUBLE), hasValue(Math.expm1(value))));
+    assertEquals(String.format("expm1(%s)", value.toString()), expm1.toString());
+  }
+
+  /**
+   * Test expm1 with null value.
+   */
+  @Test
+  public void expm1_null_value() {
```

Comment:
You can omit null/missing tests, because it is covered by `FunctionDSLnullMissingHandlingTest`
You don't actually test your function there, you test `nullMissingHandling`

Reply:
Would this mean that every function in `MathematicalFunctionTest.java` doesn't need a null/missing test? Because currently all math functions have tests for null/missing in the exact same format

Reply:
Yes I believe we do not need it, even if it exists for other functions. I believe I had similar comments on a previous PR which included tests in `DateTimeFunctionTests.java` (IIRC the behaviour for `nullMissingHandling` is the same for all tests, so it doesn't need to be tested multiple times by each function).

Reply:
I have removed the unnecessary tests for EXPM1